log/log.go

```diff
@@ -0,0 +1,192 @@
+// Copyright © 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package log
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+)
+
+const (
+	// LevelLogError specified the log level as: ERROR
+	LevelLogError = 1
```

Comment:
Let's call these `LogLevelError`, `LogLevelWarning` etc. 
Use iota + 1 to enumerate the constants. E.g.
```
// Specifies the log levels
const (
	LogLevelError = iota + 1
	LogLevelWarning
	LogLevelInfo
	LogLevelDebug
	LogLevelVerbose // This is the same as Debug, but without the repeat filtering
)
```